app/uk/gov/hmrc/economiccrimelevyregistration/views/NotLiableView.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.economiccrimelevyregistration.utils.EclTaxYear
+
+@this(
+        layout: templates.Layout
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle = title(messages("notLiable.title"))
+) {
+
+    <h1 class="govuk-heading-l">@messages("notLiable.heading")</h1>
+
+    <p class="govuk-body">@messages("notLiable.body", EclTaxYear.currentFinancialYear)</p>
+
+    <div class="govuk-inset-text">
+        <p>@messages("notLiable.inset.p1")</p>
+        <p>@Html(messages("notLiable.inset.p2", messages("notLiable.inset.link")))</p>
+    </div>
```

Comment:
There is a twirl component for inset text we should use, see: https://design-system.service.gov.uk/components/inset-text/#injected-example-play

Reply:
👍